scripts/variantstore/wdl/GvsExtractCallset.wdl

```diff
@@ -437,9 +395,9 @@ task CreateManifest {
   }
 
   runtime {
-    docker: "gcr.io/google.com/cloudsdktool/cloud-sdk:398.0.0"
+    docker: "gcr.io/google.com/cloudsdktool/cloud-sdk:404.0.0-alpine"
     memory: "3 GB"
-    disks: "local-disk 10 HDD"
+    disks: "local-disk 500 HDD"
```

Comment:
what drove the increase in the disk size?

Reply:
This was just a drive-by bump for performance improvement since I was already in there. The task itself should be very short running so hopefully this disk size bump will significantly improve overall runtime.